src/cdk/utils/basic-authorizer-handler/index.test.ts

```diff
@@ -0,0 +1,15 @@
+import {getAuthHeaderValue} from '.';
+
+describe('getAuthHeaderValue', () => {
```

Comment:
Please change to `describe('getAuthHeaderValue()', () => {`

Reply:
Fixed